src/screens/edit-profile-screen/launchSelectImageActionSheet.ts

```diff
@@ -0,0 +1,101 @@
+import { ActionSheetIOS, Alert, Platform } from 'react-native'
+import {
+  ImageLibraryOptions,
+  launchImageLibrary,
+  launchCamera,
+  Asset
+} from 'react-native-image-picker'
+
+import { Image } from './types'
+
+const photoOptions: ImageLibraryOptions = {
+  includeBase64: true,
+  maxWidth: 1440,
+  mediaType: 'photo',
+  quality: 0.9
+}
+
+export const launchSelectImageActionSheet = (
+  callback: (image: Image) => void
+) => {
+  const handlePhoto = ({ assets }: { assets: Asset[] | undefined }) => {
+    const response = assets?.[0]
+    const selectedPhoto = !!response?.base64
+    if (selectedPhoto) {
+      const image = {
+        height: response.height ?? 0,
+        width: response.width ?? 0,
+        name: response.fileName ?? response.uri?.split('/').pop() ?? '',
+        size: response.fileSize ?? 0,
+        fileType: response.type ?? '',
+        uri: response.uri ?? '',
+        file: `data:${response.type};base64,${response.base64}`
+      }
+      callback(image)
+    }
+  }
+
+  const selectPhotoFromLibrary = () => {
+    launchImageLibrary(
+      {
+        ...photoOptions,
+        selectionLimit: 1
+      },
+      handlePhoto
+    )
+  }
+
+  const takePhoto = () => {
+    launchCamera(
+      {
+        ...photoOptions,
+        saveToPhotos: true
+      },
+      handlePhoto
+    )
+  }
+
+  if (Platform.OS === 'ios') {
+    // iOS ActionSheet
+    ActionSheetIOS.showActionSheetWithOptions(
+      {
+        options: ['Cancel', 'Photo Library', 'Take Photo'],
+        tintColor: '#7E1BCC',
```

Comment:
Could this color come from the theme?

Reply:
yeah def, suppose itll be a lil hard here since this is a function though, what do you think?